extensions/amp-recaptcha-input/amp-recaptcha-input.md

```diff
@@ -78,6 +78,10 @@ grecaptcha.execute('reCAPTCHA_site_key', {action: 'reCAPTCHA_example_action'});
     <td width="40%"><strong>data-action (required)</strong></td>
     <td><a href="https://developers.google.com/recaptcha/docs/v3">reCAPTCHA v3</a> action to be executed on form submission.</td>
   </tr>
+  <tr>
+    <td width="40%"><strong>global [optional]</strong></td>
+    <td>Include this optional attribute to change the endpoint from which the iframe loads the reCAPTCHA api script.  More info can be found here: <a href="https://developers.google.com/recaptcha/docs/faq#can-i-use-recaptcha-globally">reCAPTCHA FAQ</a>.</td>
```

Comment:
From a documentation standpoint, when an attribute is optional it's important to describe what it does when it is present, and what it does by default:

`By default, the iframe loads the recaptcha api script using the 'www.google.com' endpoint. There are some situations when this is not accessible. When the "data-global" attribute is included, the component will load the script from the "www.recaptcha.net" endpoint instead. More info ... `

Reply:
Your example documentation is much more descriptive.  I've included this version with some minor edits!

Reply:
Your example documentation is much more descriptive.  I've included this version with some minor edits!